README.md

```diff
@@ -1,2 +1,12 @@
 # cloud-armor-node-controller
 Temp repo for cloud-armor-node-controller
```

Comment:
I would kill line 1-2 here, and add in an overview section here. The rest looks fine, though if you get the cloudbuild in first you could offer both instructions.

Reply:
On second thought, I would focus on just getting this README to be reasonable, then we can work on the Cloudbuild step after.